homeassistant/components/neptune_apex/sensor.py

```diff
@@ -0,0 +1,89 @@
+"""Support for getting probe data from Neptune Apex."""
```

Comment:
Please limit the first PR to one platform. More platforms can come later.

Reply:
Obviously if that's the only way to get the PR through, I will absolutely do that, but the actual code talking to the Apex makes a single call to pull both sensor data (temperature, pH, etc) and outlet state (on/off/profile). Please advise.

Reply:
It doesn't matter if we don't expose all the polled data. So we can add the sensor platform later.

Reply:
OK. Should I remove the files or just take them out of the array?